src/runtime-rs/crates/hypervisor/src/qemu/cmdline_generator.rs

```diff
@@ -2119,3 +2059,19 @@ pub fn get_network_device(
 
     Ok((netdev, virtio_net_device))
 }
+
+fn get_devno_ccw(ccw_subchannel: &mut Option<CcwSubChannel>, device_name: &str) -> Option<String> {
```

Comment:
IMO, It can be simplified further with `map` and  `and_then` combinations. 
Without this, most looks good to me.

Reply:
Oh, let me try it out. Thanks for the advice. 😉